test/unit-tests.el

```diff
@@ -0,0 +1,10 @@
+;;; unit-tests.el --- Description -*- lexical-binding: t; -*-
+;; org-transclusion unit tests
+
+(ert-deftest org-transclusion-encrypted-org-file-p-test ()
+  "Tests org-transclusion-encrypted-org-file-p-test"
+  (should (equal (org-transclusion-encrypted-org-file-p "test.org.gpg") t))
+  (should (equal (org-transclusion-encrypted-org-file-p "test.org") nil))
```

Comment:
Is it possible to merge the two functions, so that we get `t` for the second test case?

```
(should (equal (org-transclusion-encrypted-org-file-p "test.org") t))
```

Reply:
If the combined function becomes too crowded, maybe it makes more sense to separate them; but in general I would prefer to have them combined....

Reply:
Thank you for the comments. As mentioned in my other comment, I think it's fine to merge them. I'll get back to you with the merged version tomorrow.